docs/index.md

```diff
@@ -83,10 +83,16 @@ h2, h3, h4 {
   </div>
   <div class="col3rd">
     <h3>Installing Kubernetes on Linux with kubeadm</h3>
-    <p>This quickstart will show you how to install a secure Kubernetes cluster on any computers running Linux, using a tool called <code>kubeadm</code> which is part of Kubernetes. It'll work with local VMs, physical servers and/or cloud servers, either manually or as part of your own automation. It is currently in alpha but please try it out and give us feedback!</p>
+    <p>This quickstart will show you how to install a secure Kubernetes cluster on any computers running Linux, using a tool called <code>kubeadm</code>. It'll work with local VMs, physical servers and/or cloud servers, either manually or as a part of your own automation. It is currently in alpha but please try it out and give us feedback!</p>
+    <p>If you are looking for a complete production cluster, note that kubeadm is intended as a building block.  Tools such as GKE and kops build on kubeadm to provision a complete cluster.</p>
```

Comment:
My main change here was to add the second paragraph.  I called it a "building block", not a "toy", which I believe is the correct positioning.  I do not imply that kubeadm is unsafe for production.  The intention is to say "kubeadm is a building block that you can build up into a complete solution, here are two opinionated solutions if you just want a k8s cluster now".  Perhaps if I removed the word "production" from "complete production cluster" - but then "If you are looking for a complete cluster...." read awkwardly.  Equally I felt "If you are looking for a production-ready cluster" would imply that kubeadm was not production-ready.  Suggestions?

The reason the first paragraph came up in the diff is because I removed "which is part of Kubernetes", because

1) I didn't think it read very well
2) It certainly wouldn't have read very well if we then mentioned other tools.  It would beg the question "Is GKE part of Kubernetes?" which is a _real_ hairball ;-)
3) The page layout was awkward anyway.  Personally I think we'll have to shorten this whole section anyway, but I thought this would be uncontroversial...

If you do care, I can put back "which is part of Kubernetes"